libultraship/libultraship/ConfigFile.h

```diff
@@ -29,9 +29,9 @@ namespace Ship {
 			bool CreateDefaultConfig();
 
 		private:
-			mINI::INIFile File;
 			mINI::INIStructure Val;
 			std::weak_ptr<GlobalCtx2> Context;
 			std::string Path;
+			mINI::INIFile File;
 	};
```

Comment:
Any reason for moving this? This diff is pretty big, so I'm wondering if it can be distilled.

Reply:
Unordered member instantiation in a class constructor makes GCC output a warning because c++ does not respect this order. c.f : "Initialization order",  https://en.cppreference.com/w/cpp/language/constructor